R/calcEmissions.R

```diff
@@ -1127,8 +1127,8 @@ calcEmissions <- function(datasource = "CEDS16") {
     }
 
     tmp <- tmp * 44 / 12 / 1000 # from ktC -> MtCO2
-
```

Comment:
This is the only time when I dropped data before 1970 from historical.mif, because the description below says so